pkg/otlp/model/attributes/azure/azure.go

```diff
@@ -55,8 +55,7 @@ func HostnameFromAttributes(attrs pdata.AttributeMap) (string, bool) {
 
 // ClusterNameFromAttributes gets the Azure cluster name from attributes
 func ClusterNameFromAttributes(attrs pdata.AttributeMap) (string, bool) {
-	// Get cluster name from resource group
-	// https://github.com/DataDog/datadog-agent/blob/aad29b8/pkg/util/azure/azure.go#L51
+	// Get cluster name from resource group from pkg/util/cloudprovider/azure:GetClusterName
```

Comment:
I would revert this change, so that the comment reflects the version we used when writing this code

Reply:
What the point of this comment ? I thought it was to reference the implementation. What the point of pointing to a piece of code that no longer exists ?

Reply:
You are right, the implementation should not have breaking changes, approving from my team :)